randovania/games/fusion/logic_database/Sector 3 PYR.txt

```diff
@@ -267,7 +273,9 @@ Extra - room_id: [6, 24]
           Can Use Bombs
           Can Use Power Bombs and Can Use Springball
   > Other to Namihe's Abode
-      Can Use Any Bombs
+      Any of the following:
+          Can Use Any Bombs
+          Morph Ball and Screw Attack and Knowledge (Beginner)
```

Comment:
this can be simplified to 
all of
  morph ball and can break single bomb block

Reply:
Picked up change.